components/hubspot/actions/create-engagement/create-engagement.mjs

```diff
@@ -37,24 +41,95 @@ export default {
         },
       ],
     },
+    toObjectType: {
+      propDefinition: [
+        common.props.hubspot,
+        "objectType",
+      ],
+      label: "Associated Object Type",
+      description: "Type of object the engagement is being associated with",
```

Comment:
```suggestion
      description: "Type of object the engagement is associated with",
```

Reply:
Changed "being associated with" to "associated with" to follow Google's Developer docs style guide (https://developers.google.com/style/voice)